client/app/routes/home.tsx

```
@@ -0,0 +1,13 @@
+import { Link } from "react-router";
+import type { Route } from "./+types/home";
+
+export function meta({}: Route.MetaArgs) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)